q2_diversity/_beta/_visualizer.py

```diff
@@ -140,7 +140,9 @@ def beta_group_significance(output_dir: str,
         group_distances, x_ticklabels = \
             _get_distance_boxplot_data(distance_matrix, group_id, groupings)
 
-        ax = sns.boxplot(data=group_distances)
+        ax = sns.boxplot(data=group_distances,
+                         flierprops={'marker': 'o', 'markerfacecolor': 'blue',
```

Comment:
We have box plots generated by both [seaborn](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2.0.6%2Fdata%2Ftutorials%2Fmoving-pictures%2Fcm1441%2Funweighted-unifrac-body-site-significance.qzv) and by [d3](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2.0.6%2Fdata%2Ftutorials%2Fmoving-pictures%2Fcm1441%2Fevenness-group-significance.qzv) --- we probably want to keep the styles as consistent as we can between the two.

As far as style, how about we go with a black filled-in circle, about the same size as the one in the d3 plots?